ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -215,6 +215,13 @@
   public static final String KSQL_QUERY_PULL_MAX_QPS_DOC = "The maximum qps allowed for pull "
       + "queries. Once the limit is hit, queries will fail immediately";
 
+  public static final String KSQL_QUERY_PULL_SET_REPLYING_HOST_CONFIG =
```

Comment:
I feel like it would be better to add a keyword to the query itself, like `DEBUG` instead of using a server config. This way, we have a control know per pull query and we can add more debug information in the future, like amount of lag, or routing information, etc.